.github/workflows/load_test.yml

```
@@ -0,0 +1,38 @@
+on:
+  release:
+    types:
+      - prereleased
+  pull_request:
+    branches: [ feat/automatizeLoadTests, main ]
+
+name: Running Load Tests
+jobs:
+  load_tests:
+    name: unning Load Tests
+    runs-on: ubuntu-latest
```

`setup-minikube` seems to be quite coupled to the ubuntu version ([see readme](https://github.com/manusa/actions-setup-minikube/#setup-minikube-github-action)). I'd suggest to pin this to `20.04` (which is supported in `2.2.0`)
```suggestion
    runs-on: ubuntu-20.04
```

---

👍